Wrappers/Python/ccpi/web_viewer/test/test_trameviewer3d.py

```diff
@@ -483,15 +483,6 @@ def test_load_file_updates_windowing_defaults(self, _):
 
         self.trame_viewer.update_windowing_defaults.assert_called_once_with(windowing_method)
 
-    @mock.patch("ccpi.web_viewer.trame_viewer.TrameViewer.load_file")
-    def test_load_file_saves_a_new_camera_position(self, _):
-        cam_data = mock.MagicMock()
-        self.trame_viewer.original_cam_data = cam_data
-
-        self.trame_viewer.load_file("path")
-
-        self.assertNotEqual(self.trame_viewer.original_cam_data, cam_data)
-
```

Comment:
Some of the unit tests have been removed because the variables they were referring to in the trame viewer no longer exist.

For example, this unit test got deleted because we no longer save the camera position in the trame viewer. But I could change it so that it tests the equivalent variables in the CILViewer.